iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/DanglingLineXmlTest.java

```diff
@@ -7,25 +7,19 @@
 
 package com.powsybl.iidm.xml;
 
-import com.powsybl.commons.AbstractConverterTest;
-import com.powsybl.iidm.network.Network;
 import org.junit.Test;
 
 import java.io.IOException;
 
+import static com.powsybl.iidm.xml.IidmXmlTestConstants.*;
+
 /**
  * @author Mathieu Bague <mathieu.bague@rte-france.com>
  */
-public class DanglingLineXmlTest extends AbstractConverterTest {
+public class DanglingLineXmlTest extends AbstractXmlConverterTest {
 
     @Test
     public void test() throws IOException {
-        Network network = NetworkXml.read(getClass().getResourceAsStream("/danglingLine.xml"));
-
-        roundTripXmlTest(network,
-                NetworkXml::writeAndValidate,
-                NetworkXml::read,
-                "/danglingLine.xml");
-
+        roundTripVersionnedXmlTest("danglingLine.xml", IIDM_VERSION_1_0_DIR_NAME, IIDM_CURRENT_VERSION_DIR_NAME);
```

Comment:
It's not really automatic to do it like this... Everytime a version of XIIDM is released, you will have to add a constant here.

Reply:
It was on purpose: imagine an equipment is created on 1.2 and we are in 1.4. We don’t want to test backward compatibility for all versions, only 1.2 and 1.3. We could maybe write a method for all versions newer than a given version? But what if we delete a component but still want to test it is correctly interpreted (which can be for example the case for current limits) ?